src/apis/donationApi.js

```diff
@@ -0,0 +1,32 @@
+import { BASE_URL } from "../constants/constants";
+
+export async function getdonationList(cursor, pageSize = 10, priorityIdolIds) {
+  let url=`${BASE_URL}/donations`
```

Comment:
진짜 아무짝에 쓸모없는 사소한거긴한데..
```suggestion
  let url =`${BASE_URL}/donations`
```
이렇게 하시는거 어떨까요.. 통일했으면좋겠어요..!ㅎㅎㅎ

Reply:
넵! 죄송합니다ㅎㅎ..